### DISCUSSION TITLE
CUY UNIVERS semester 4 | sesi4

### DISCUSSION THREAD

**Discussion by Destaanugra1** [deaafrizal/forum#47] [Troubleshooting Room]
![Screenshot (135)](https://github.com/deaaprizal/forum/assets/152362800/655042bf-bdd9-4546-97d3-f345e5476bdc)
![Screenshot (137)](https://github.com/deaaprizal/forum/assets/152362800/ee7c4d9a-1abd-4ab3-a26c-2c458bf29e2e)
![Screenshot (134)](https://github.com/deaaprizal/forum/assets/152362800/d3e6e5da-f7ce-41fb-b748-8ebee
![Screenshot (133)](https://github.com/deaaprizal/forum/assets/152362800/8d5fea56-71cd-4212-a5ee-d92260e31f99)
a73ed7f)
kenapa pas link nya populer tapi di klik kok palah notfound
maaf sebelum banyak tanya karena baru pertama megang react

**Comment by deaafrizal** [comment]
cek huruf besar kecil nama file nya bro, itu ngaruh juga.
